apps/openmw/mwrender/sky.cpp

```diff
@@ -557,27 +558,33 @@ class Sun : public CelestialBody
     {
         osg::ref_ptr<osg::OcclusionQueryNode> oqn = new osg::OcclusionQueryNode;
         oqn->setQueriesEnabled(true);
-
+#if OSG_VERSION_LESS_THAN(3,6,5)
+        osg::Geometry* queryGeom = oqn->getQueryGeometry();
```

Comment:
Always `ref_ptr` OSG stuff. Even if it's perfectly safe here, it's safer if other people start changing things later.